deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: aws.managed.openshift.io/v1alpha1
 kind: AWSFederatedRole
 metadata:
-  name: customer-admin
+  name: network-mgmt
   namespace: aws-account-operator
 spec:
-  roleDisplayName: customer-admin
-  roleDescription: This is the description of customer-admin
+  roleDisplayName: Network Management
```

Comment:
"Network management"

Reply:
Is it a UI requirement for these to be quoted?

Reply:
No No.
I meant `Network management`.
see https://marvelapp.com/d780h9h/screen/64100971

Reply:
Oh I see now. Will fix